assets/prototype/domain/eventEditor/EventEditor.tsx

```diff
@@ -7,8 +7,8 @@ import * as React from 'react';
  * Internal dependencies
  */
 import useInitQueries from './data/initialization/useInitQueries';
-import DatesList from './datetimes/DatesList';
-import TicketsList from './tickets/TicketsList';
+import DatesList from './datetimes/datesList';
+import TicketsList from './tickets/ticketsList';
```

Comment:
@tn3rb I did not fully understood what was the expected result here.
Is this ok?

Reply:
I did not understood this part:
`delete \domain\eventEditor\datetimes\datesList`
and
`delete \domain\eventEditor\tickets\ticketsList`

Reply:
there were two folders `datesList` and `DatesList` with the files in the latter, but we want to use the first name for the folder because `DatesList` implies a component.

I could have also said to move contents of `DatesList` into `datesList` and then delete `DatesList`.